### DISCUSSION TITLE
Entregable Modulo 10

### DISCUSSION THREAD

**Discussion by alangtzod** [LaunchX-InnovaccionVirtual/CursoIntroPython#283] [Q&A] [answered]
Hola, que se entrega en el modulo 10 como Kata???

**Comment by codingcodax** [comment, marked as answer]
Tienes que recrear los ejercicios que vienen en el Markdown.
Es un archivo o varios si quieres de python. Ejemplo: `open.py`.

**Reply by NatanDelgado** [reply]
una duda aqui que se hace??? ya intente creando una carpeta que se llame config.txt pero no me sale el error que dice la explicacion

![image](https://user-images.githubusercontent.com/88174320/154788787-9640806d-46b3-48b6-8baf-94c14bc65583.png)



sabras porque pasa eso???

**Comment by ghost** [comment]
segun lei en otras respuestas unos lo hicieron con capturas de pantalla, asi que toca hacer los ejercicios tomando captura del codigo en vs y el error en la terminal de vs, pero nose si se los aceptaron asi .-.

**Reply by Gracesl** [reply]
La kata 10 me esta causando dolor de cabeza

**Comment by PARZANTONIO** [comment]
(Ayuda)
Recreando los problemas, el último problema: Los litros de agua para los astronautas.
el archivo .py no ejecuta lo que se menciona en el markdown. Realizando los ejercicios desde VSC con archivos .py 
En cuanto:
Para el ejercicio de los archivos donde se mencionan permisos, si los realizan mediante VSC o Phyton el archivo creado siempre tendrá permisos 777, no les arrojará errores al menos que manualmente le quiten los permisos al archivo/carpeta (en mi caso)
